bin/trivy-scan.sh

```diff
@@ -38,4 +38,6 @@ if grep -q -i 'OS is not detected' trivy-output.txt ; then
   exit 0
 fi
 
-exit "${exit_code}"
+#exit "${exit_code}"
+export PROJECT_DIRECTORY="${scan_target}"
+python bin/trivy_validator.py
```

Comment:
I see that trivy_validator.py exits non-zero if there are CVEs, but I think we should keep this logic in as a fallback. It doesn't hurt.

```suggestion
export PROJECT_DIRECTORY="${scan_target}"
python bin/trivy_validator.py
exit "${exit_code}"
```